compute_endpoint/globus_compute_endpoint/engines/high_throughput/engine.py

```diff
@@ -431,7 +434,7 @@ def _start_local_interchange_process(self):
             name="Engine-Interchange",
             args=(comm_q,),
             kwargs={
-                "client_address": "127.0.0.1",  # engine and ix are on the same node
+                "client_address": "localhost",  # engine and ix are on same node
```

Comment:
I'm not looking now, but does the other end of this require an IP address like this `__init__` method has?  If so, we need to be sure that `localhost` works for it as well.

Reply:
Apparently not?  Localhost seems to work.  ie. this is how it's used, similar to ip_address:

        self.task_incoming.connect(f"tcp://{client_address}:{client_ports[0]}")

Reply:
We just need to make sure that IPv6 is specified to the worker managers as well.  Otherwise, without an explicit IP (v6 or v4) address, I suspect we'll get seemingly random mismatches going forward.

Reply:
We saw that localhost works in parsl, I think this is ok, right?  per earlier debugging session.